examples/Finding Pipelines with Available Updates.ipynb

```diff
@@ -0,0 +1,790 @@
+    "\n",
+    "The Protocols endpoint also returns a lot of information. We'll again subset to only get the info we need.\n",
+    "\n",
+    "- `namespace` - `common`, `client` or `private` depending on how the protocol was deployed\n",
+    "- `slug` - The internal unique name for the protocol\n",
+    "- `version` - The version we can compare to the latest versions\n",
+    "- `name` - The display title for the protocol\n",
+    "- `description` - Additional information about the function of the protocol"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "f1b3d540",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "PROTOCOL_COLUMNS = [\n",
+    "    \"namespace\",\n",
+    "    \"slug\",\n",
+    "    \"version\",\n",
+    "    \"name\",\n",
+    "    \"description\",\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "1ad57f7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "protocol_df = pd.json_normalize(latest_protocols)[PROTOCOL_COLUMNS]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "61ba6f63",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>namespace</th>\n",
+       "      <th>slug</th>\n",
+       "      <th>version</th>\n",
+       "      <th>name</th>\n",
+       "      <th>description</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>client-merck</td>\n",
+       "      <td>create-empower-ssm-and-send-to-agent</td>\n",
+       "      <td>v0.1.0</td>\n",
+       "      <td>Create Empower SSM And Send To Agent</td>\n",
+       "      <td>Create a SSM to send to Empower</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>common</td>\n",
+       "      <td>thermofisher-xcalibur-raw-to-ids</td>\n",
+       "      <td>v4.0.3</td>\n",
+       "      <td>Thermo Fisher Xcalibur Raw To IDS</td>\n",
+       "      <td>Convert Thermo Fisher Xcalibur RAW files to IDS</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      namespace                                  slug version   \n",
+       "0  client-merck  create-empower-ssm-and-send-to-agent  v0.1.0  \\\n",
```

Comment:
same here - we can't use customer name in the tables. can you query the tables to get clean data (not use .head())?

Reply:
Yeah was thinking about that... I was hoping we'd have a clean environment to run this in, but those seem hard to find. At least this was a lot cleaner than my personal environment....

I'll make an update to ensure none of them show